js/constants.js

```diff
@@ -57,8 +57,11 @@ window.constants = (function () {
   var PIN_HEIGHT = 70;
   var MAIN_PIN_WIDTH = 62;
   var MAIN_PIN_HEIGHT = 84;
+  var MAIN_PIN_DEFAULT_X = 570;
+  var MAIN_PIN_DEFAULT_Y = 375;
 
   var LOAD_URL = 'https://javascript.pages.academy/keksobooking/data';
+  var UPLOAD_URL = 'https://javascript.pages.academy/keksobooking';
```

Comment:
Такое себе, а если будут ещё запросы, под каждый идентичную константу ?
```suggestion
var API_URL = = 'https://javascript.pages.academy/keksobooking/';
```

А в методе уже 
```
     xhr.open('POST', window.constants.API_URL + 'data'); 
     xhr.open('GET', window.constants.API_URL); 
```